mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -176,12 +176,12 @@ def start_app(self, wait_time=APP_START_WAIT_TIME):
             '%s failed to start on %s.' % (self.app_name, self._adb.serial))
 
     def connect(self, port, addr='localhost', uid=UNKNOWN_UID,
-                connection_timeout=None, cmd=JsonRpcCommand.INIT):
+                cmd=JsonRpcCommand.INIT):
```

Comment:
We only make one call to this function and that call dose not use this param.
So there's really no point in having it for practical purposes.